buildlib/azure-pipelines-pr.yml

```diff
@@ -23,6 +23,8 @@ pr:
     - buildlib/azure-pipelines-perf.yml
     - buildlib/tools/perf_results.py
     - buildlib/tools/perf-common.yml
+    - buildlib/azure-pipelines-release.yml
+    - buildlib/jucx/jucx-build.yml
```

Comment:
but it's also used for PR that has the title AZP/RELEASE

Reply:
In this case, we can't exclude any of the release-related files from the PR pipeline. 
Rolling back.